xicam/catalog_viewer/__init__.py

```diff
@@ -21,11 +21,11 @@ def __init__(self):
         self.parent_widget = QWidget()
         # add combobox to select stream
         self.field_label_stream = QLabel('Select stream:')
```

Comment:
should we make it consistent and call it stream_label?
```suggestion
        self.stream_label = QLabel('Select stream:')
```